README.md

```diff
@@ -16,6 +16,9 @@ The recommended way to install composer packages is:
 
 ```
 $ composer require bakkerij/cakeadmin:dev-rewrite
+$ composer require friendsofcake/crud:^4.3
+$ composer require gourmet/knp-menu:~0.4
+$ composer require holt59/cakephp3-bootstrap-helpers:dev-master
```

Comment:
This is because you added the plugin to your `plugins/Bakkerij/CakeAdmin` for development reasons. Composer won't get that file. 

When the package is loaded by an user, the package is located in the `vendor` folder, and Composer will recognize it, and load its dependencies.

TL;DR: This is only needed for development reasons but it might be solved another way...